extensions/ql-vscode/src/query-history/store/data-mapper.ts

```diff
@@ -54,6 +64,16 @@ function mapLocalQueryDataItemToDomainModel(
 function mapCompletedQueryInfoDataToDomainModel(
   completedQuery: CompletedQueryInfoDto,
 ): CompletedQueryInfo {
+  const sortState =
```

Comment:
Not that this mapper looks wrong, but why does renaming the `SortDirectionDto` type mean we now have to provide a mapper? The type itself hasn't changed but just its name. It doesn't seem like we were accidentally using a DTO type where we shouldn't have been, or accidentally referencing a domain type from the DTO type.

Is this some crazy enum magic, where you can cast them to each other if they are defined in separate files but have the same name?